examples/plot_dcrt_example.py

```diff
@@ -52,16 +51,18 @@
     y = np.maximum(0.0, y)
 
     ## dcrt Lasso ##
-    results_lasso = dcrt_zero(X, y, screening=False, verbose=True)
+    selection_features, X_res, sigma2_X, y_res = dcrt_zero(X, y, screening=False)
+    results_lasso = dcrt_pvalue(selection_features, X_res, sigma2_X, y_res)
```

Comment:
Since the second function only uses as arguments the outputs of the first, it could actually call it as a sub-routine ? I don't find the current schema very nice.

Reply:
I made a separation between the output of the function and the pvalue because some other methods don't require the computation of the pvalue. 
Moreover, with this intermediate result, it's possible to compute the confidence interval instead of the pvalue, as you seem to want to do it in the future. 

I don't really how to use a subroutine because you don't want to have a different number of arguments in the output of one function. In consequence, I will need to increase the amount of output elements to 7, which starts to be a lot. 
Another way should be to use a class for tracking the internal state and avoid passing the value from one function to another.

Reply:
But the rationale of dcrt is really to get p-values: it's a feature importance analysis method.  If you only want to select or rank variables, there are many other things to do.

I take your point on a class, but i think that the first question is to think stronger about use of the method to make sure that we do it right (I'm still not convinced that we do)

Reply:
If this method is a feature importance analysis, I don't understand why it also provides some "variable importance"? And why it doesn't require some feature importance list as an input of the method?